ibrahim/codepen.html

```diff
@@ -0,0 +1,170 @@
+<!-- Copied from https://codepen.io/ibnoon/pen/MWwqYJQ?editors=1000  -->
+
+<!-- MAX.G Logo -->
+<img class="MaxG" src="http://angeliqueweger.com/temp/gd2-images/ibrahim/inoon_small_icon.svg" alt="Max Gallery">
+
+<img class="MaxG_l" src="http://angeliqueweger.com/temp/gd2-images/ibrahim/inoon_banner.svg" alt="Max Gallery">
+
+<!-- Nav -->
+<nav>
+  <ul class="nav">
+    <li><a href="#">About</a></li>
+    <li><a href="#">Event</a></li>
+    <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
+
+<nav>
+  <ul class="nav_l">
+    <li><a href="#">About</a></li>
+    <li><a href="#">News</a></l>
+    <li><a href="#">Event</a></li>
+    <li><a href="#">Exhibition</a></li>
+    <li><a href="#">Publications</a></li>
+    <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
+
+<!-- Header Image -->
+<img src="http://angeliqueweger.com/temp/gd2-images/gallery3.jpg" alt="">
+
+<div class="body">
+
+  <div class="text">
+    <ul class="text-grid">
+
+      <li>
+        <h2>Dance of Conversation</h2>
+        <p>Friday March 6, 2020 - Saturday, March 28, 2020
+          Opening Reception: Saturday, March 14, 2020</p>
+      </li>
+
+      <li>
+        <h2>Featured Artists</h2>
+        <p>Shelley Amsel, Jude Asher, Toni Berger, Mary Bickford, Joel Cohen, Diane Dennis, Landis Expandis, Jackie Mintz, Michael Tice, and Carlos Walker</p>
+      </li>
+
+    </ul>
+  </div>
+
+  <div class="main-content">
```

Comment:
I don't think you need this `div` -- the only styles I notice on it are the `overflow-x`, which I don't think is accomplishing anything and a `padding-bottom`, which can be moved to another element instead?